pandas/core/arrays/sparse/accessor.py

```diff
@@ -119,24 +120,58 @@ def to_coo(self, row_levels=(0,), column_levels=(1,), sort_labels=False):
         Examples
         --------
         >>> s = pd.Series([3.0, np.nan, 1.0, 3.0, np.nan, np.nan])
-        >>> s.index = pd.MultiIndex.from_tuples([(1, 2, 'a', 0),
-                                                (1, 2, 'a', 1),
-                                                (1, 1, 'b', 0),
-                                                (1, 1, 'b', 1),
-                                                (2, 1, 'b', 0),
-                                                (2, 1, 'b', 1)],
-                                                names=['A', 'B', 'C', 'D'])
+        >>> s
+        0    3.0
+        1    NaN
+        2    1.0
+        3    3.0
+        4    NaN
+        5    NaN
+        dtype: float64
```

Comment:
I would leave this out, since you already show the output of `s` below after setting the index, and it is the Series with the multi-index that is relevant for the rest of the example